src/main/scala/com/lucidchart/android/javascript/package.scala

```diff
@@ -23,7 +29,7 @@ package object javascript {
 
   implicit def toJsParameterValue[A: JsParameter](a: A): JsParameterValue = {
     new JsParameterValue {
-      override def toString: String = implicitly[JsParameter[A]].asJsString(a)
+      override val toString: String = implicitly[JsParameter[A]].asJsString(a)
```

Comment:
why val and not a def?

Reply:
Gregg

Reply:
it's so you only evaluate the string once ever instead of everytime you call it